frontend/app/src/components/history/filtering/TableFilter.vue

```diff
@@ -249,4 +252,23 @@ const moveSuggestion = (up: boolean) => {
     set(selectedSuggestion, position);
   }
 };
+
+const getDisplayValue = (suggestion: Suggestion) => {
+  let value = suggestion.value;
+  if (typeof value === 'string') {
+    return value;
+  }
+
+  return value.isCustomAsset ? value.name : value.symbol;
+};
+
+const getSuggestionText = (suggestion: Suggestion) => {
+  return `${suggestion.key}: ${getDisplayValue(suggestion)}`;
+};
```

Comment:
This feels like the wrong place to have this code. It is too specific to the custom assets. We need a different approach here.

Let me think about it a bit.

Reply:
Since this will require some rework can you add a `//TODO: remove it after introducing a display value mechanism`